### PR TITLE
Add, remove and update icons

### LINKED ISSUE #11088: Update icon designs, remove Major and Minor suffix, and update icon names
### Why are you proposing this update?

A few reasons:
- Some icons in the nav have an odd visual shift between filled and outline, so we reduced it as much as possible.
- A few metaphors being used weren't right, so we updated it.
- Lastly, we took the opportunity to clean up/improve a couple icons that needed it.

### Icon name

See description above

### What type of icon set?

Major (20x20)

### Icon SVG code

[Icons Update.zip](https://github.com/Shopify/polaris/files/13243684/Icons.Update.zip)

### Release strategy

- [x] https://github.com/Shopify/polaris-internal/issues/1177
- [x] Migrate and update web deps
  - [x] `@shopify/app-bridge-host` (NA)
  - [x] `@shopify/app-kit-internal` https://github.com/Shopify/app-kit/pull/624
  - [x] `@shopify/flow-app` https://github.com/Shopify/flow-app/pull/3113
  - [x] `@shopify/magic-ui` https://github.com/Shopify/magic-ui/pull/202
  - [x] `@shopify/online-store-ui` https://github.com/Shopify/online-store-ui/pull/4382
- [x] https://github.com/Shopify/web/pull/114592
- [x] Update [store activity icon](https://github.com/Shopify/polaris-internal/issues/1196#issuecomment-1843091677)

## Post Release

- [ ] Share post detailing icon release stats
  - [ ] Reduction is total icons in the system
  - [ ] Smaller library bundle size
  - [ ] Reduced number of icons used in the admin
  - [ ] Any other metrics worth mentioning?

### Icon updates

- [x] Removed Major and Minor icon sizing. All icons are the same size.
- [x] Updated icon designs (see SVG zip)
- [x] Added icons `ShippingLabelIcon`, `ShippingLabelFilledIcon`, `BillIcon`, `BillFilledIcon`, `ActivityLogIcon`, `ActivityLogFilledIcon`, `InstagramIcon`, `TipsIcon`
- [x] Removed icons `ReportFilledMinor`, `ReportMinor`, `ListMajor`, `ListFilledMajor`
- [x] Removed `version` and `set` from icon `.yml` files
- [x] Updated default icon color to `#4A4A4A`
- [x] Renamed many icons and replaced `Major` and `Minor` suffixes with `Icon`

### PR DESCRIPTION
**Pull request by alex-page**
### WHY are these changes introduced?

Brings in the new SVG files from https://github.com/Shopify/polaris/issues/11088.

- [x] Only uses the major icons and does not include minor icons
- [x] Runs svgo to optimize the svg files
- [x] Removes `fill="none"` from `<svg>` tags
- [x] Removes `clip-rule="` on `<path>` tags
- [x] Replaces #4A4A4A with #5C5F62 for the fill color of icons

### WHAT is this pull request doing?

Update icons for `Jobs`, `JobsFilled`, `Automation`, `AutomationFilled`, `GiftCard`, `InventoryFilled`, `Reports`, `ShipmentFilled`, `AppsFilled`, `Markets`, `MarketsFilled`, `BillingStatementDollarFilled`, `BillingStatementEuroFilled`, `BillingStatementPoundFilled`, `BillingStatementRupeeFilled`, `BillingStatementYenFilled`, `LegalFilled`, `IdentityCard`, `IdentityCardFilled`, `Domains`, `DomainNew`, `DomainsFilled`, `Store`, `OnlineStore`, `ManagedStore`, `ImportStore`, `Tax`, `TaxFilled`, `PackageFilled`, `Location`, `LocationFilled`, `InactiveLocation`, `Pin`, `Globe`, `Transport`, `Nature`, `Food`, `Colors` to use the new design language

Added new icons for `ShippingLabel`, `ShippingLabelFilled`, `Bill`, `BillFilled`, `UsersPermissions`, `UsersPermissionsFilled`, `ActivityLog`, `ActivityLogFilled`, `Instagram`, `Tips`

Removed icons `ReportFilledMinor`, `ReportMinor`, `ListMajor`, `ListFilledMajor`

### CODE REVIEW

File: polaris-icons/icons/ActivityLog.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Activity log
+description: N/A
+keywords:
+  - Page
+  - Log
+  - Activity
+authors:
+  - José Torre
+version: 1
```
Comment (sam-b-rose) [inline polaris-icons/icons/ActivityLog.yml]: If we do chose to remove the version (#11127), we should remember to update these here as well 👍

**Review by sam-b-rose — APPROVED** [review]

File: polaris-icons/icons/ActivityLog.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Activity log
+description: N/A
+keywords:
```
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLog.yml:3]: History
Timeline
Events
Past
Monitoring

File: polaris-icons/icons/ActivityLogFilled.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: Activity log filled
+description: N/A
+keywords:
```
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLogFilled.yml:3]: History
Timeline
Events
Past
Monitoring

File: polaris-icons/icons/ActivityLog.yml (hunk 2 above)
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLog.yml:3]: History
Timeline
Events
Past
Monitoring

File: polaris-icons/icons/ActivityLogFilled.yml (hunk 3 above)
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLogFilled.yml:3]: History
Timeline
Events
Past
Monitoring

File: polaris-icons/icons/ActivityLog.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  - Page
+  - Log
+  - Activity
+authors:
```
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLog.yml:7]: + Holly Schofield

File: polaris-icons/icons/ActivityLogFilled.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  - Page
+  - Log
+  - Activity
+authors:
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLogFilled.yml:8]: + Holly Schofield

File: polaris-icons/icons/AppsFilled.yml [hunk 6]
```diff
@@ -9,7 +9,8 @@ keywords:
   - extensions
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/AppsFilled.yml:12]: + Holly Schofield

File: polaris-icons/icons/ActivityLog.yml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+name: Activity log
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLog.yml:2]: Used to represent a story activity log, or any other report/series of events that happened in the past.

File: polaris-icons/icons/ActivityLogFilled.yml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+name: Activity log filled
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ActivityLogFilled.yml:2]: Used to represent a story activity log, or any other report/series of events that happened in the past.

File: polaris-icons/icons/Bill.yml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+name: Bill
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Bill.yml:2]: User to represent Bill Pay, or a receipt.

File: polaris-icons/icons/BillFilled.yml [hunk 10]
```diff
@@ -0,0 +1,10 @@
+name: Bill filled
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillFilled.yml:2]: User to represent Bill Pay, or a receipt.

File: polaris-icons/icons/Bill.yml [hunk 11]
```diff
@@ -0,0 +1,10 @@
...
+keywords:
+  - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Bill.yml:4]: - Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillFilled.yml [hunk 12]
```diff
@@ -0,0 +1,10 @@
...
+keywords:
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillFilled.yml:3]: - Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillingStatementDollarFilled.yml [hunk 13]
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementDollarFilled.yml:4]: - Dollar
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillingStatementEuroFilled.yml [hunk 14]
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementEuroFilled.yml:4]: - Euro
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillingStatementPoundFilled.yml [hunk 15]
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementPoundFilled.yml:4]: - Pound
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillingStatementRupeeFilled.yml [hunk 16]
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementRupeeFilled.yml:4]: - Rupee
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillingStatementYenFilled.yml [hunk 17]
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementYenFilled.yml:4]: - Yen
- Receipt
- Invoice
- Bill
- Payment
- Receipt
- Purchase
- Transaction
- Proof
- Expense
- Statement

File: polaris-icons/icons/BillingStatementDollarFilled.yml [hunk 18]
```diff
@@ -4,7 +4,8 @@ keywords:
...
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementDollarFilled.yml:7]: Holly Schofield

File: polaris-icons/icons/BillingStatementEuroFilled.yml [hunk 19]
```diff
@@ -4,7 +4,8 @@ keywords:
...
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementEuroFilled.yml:7]: Holly Schofield

File: polaris-icons/icons/BillingStatementPoundFilled.yml [hunk 20]
```diff
@@ -4,7 +4,8 @@ keywords:
...
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementPoundFilled.yml:7]: Holly Schofield

File: polaris-icons/icons/BillingStatementRupeeFilled.yml [hunk 21]
```diff
@@ -4,7 +4,8 @@ keywords:
...
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementRupeeFilled.yml:7]: Holly Schofield

File: polaris-icons/icons/BillingStatementYenFilled.yml [hunk 22]
```diff
@@ -4,7 +4,8 @@ keywords:
...
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/BillingStatementYenFilled.yml:7]: Holly Schofield

File: polaris-icons/icons/ColorNone.yml [hunk 23]
```diff
@@ -14,7 +14,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/ColorNone.yml:17]: Holly Schofield

File: polaris-icons/icons/DomainNew.yml [hunk 24]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
 authors:
-  - N/A
+  - José Torre
   - Joe Thomas
```
Comment (zecarlostorre) [inline polaris-icons/icons/DomainNew.yml:7]: Holly Schofield

File: polaris-icons/icons/DomainNew.yml [hunk 25]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
```
Comment (zecarlostorre) [inline polaris-icons/icons/DomainNew.yml:3]: - Register
- Add
- Domain
- Web
- Create
- Website
- Digital

File: polaris-icons/icons/DomainRedirect.yml [hunk 26]
```diff
@@ -6,7 +6,8 @@ keywords:
 authors:
   - Kyryll Odobetskiy
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/DomainRedirect.yml:9]: Holly Schofield

File: polaris-icons/icons/Domains.yml [hunk 27]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
 authors:
-  - N/A
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/Domains.yml:6]: Holly Schofield

File: polaris-icons/icons/Domains.yml [hunk 28]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Domains.yml:4]: Domain
Web
Website
Digital

File: polaris-icons/icons/DomainsFilled.yml [hunk 29]
```diff
@@ -4,7 +4,8 @@ keywords:
   - discount
```
Comment (zecarlostorre) [inline polaris-icons/icons/DomainsFilled.yml:4]: Remove:
- Discount

Add:
- Domain
- Web
- Website
- Digital

File: polaris-icons/icons/DomainsFilled.yml [hunk 30]
```diff
@@ -4,7 +4,8 @@ keywords:
...
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/DomainsFilled.yml:7]: Holly Schofield

File: polaris-icons/icons/GiftCard.yml [hunk 31]
```diff
@@ -7,7 +7,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/GiftCard.yml:10]: Holly Schofield

File: polaris-icons/icons/GiftCardFilled.yml [hunk 32]
```diff
@@ -6,7 +6,8 @@ keywords:
   - present
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/GiftCardFilled.yml:9]: Holly Schofield

File: polaris-icons/icons/Globe.yml [hunk 33]
```diff
@@ -5,7 +5,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/Globe.yml:8]: Holly Schofield

File: polaris-icons/icons/ImportStore.yml [hunk 34]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ImportStore.yml:4]: Import
Store

File: polaris-icons/icons/InactiveLocation.yml [hunk 35]
```diff
@@ -8,7 +8,8 @@ authors:
   - Shanique Shields
   - James Findlater
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/InactiveLocation.yml:11]: Holly Schofield

File: polaris-icons/icons/Jobs.yml [hunk 36]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Jobs.yml:4]: Job
Briefcase
Work
Luggage
Business

File: polaris-icons/icons/JobsFilled.yml [hunk 37]
```diff
@@ -4,7 +4,8 @@ keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/JobsFilled.yml:4]: Job
Briefcase
Work
Luggage
Business

File: polaris-icons/icons/Location.yml [hunk 38]
```diff
@@ -6,7 +6,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/Location.yml:9]: Holly Schofield

File: polaris-icons/icons/LocationFilled.yml [hunk 39]
```diff
@@ -5,7 +5,8 @@ keywords:
   - pin
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/LocationFilled.yml:8]: Holly Schofield

File: polaris-icons/icons/Markets.yml [hunk 40]
```diff
@@ -8,7 +8,8 @@ keywords:
 authors:
   - Justina Chua
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/Markets.yml:11]: Holly Schofield

File: polaris-icons/icons/MarketsFilled.yml [hunk 41]
```diff
@@ -7,7 +7,8 @@ keywords:
   - global
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/MarketsFilled.yml:10]: Holly Schofield

File: polaris-icons/icons/Nature.yml [hunk 42]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Nature.yml:4]: Nature
Leaf

File: polaris-icons/icons/PackageFilled.yml [hunk 43]
```diff
@@ -7,7 +7,8 @@ keywords:
   - cardboard
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/PackageFilled.yml:10]: Holly Schofield

File: polaris-icons/icons/PasskeyFilled.yml [hunk 44]
```diff
@@ -6,7 +6,8 @@ keywords:
   - security
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/PasskeyFilled.yml:9]: Holly Schofield

File: polaris-icons/icons/ProductsFilled.yml [hunk 45]
```diff
@@ -6,7 +6,8 @@ keywords:
   - price
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/ProductsFilled.yml:9]: Holly Schofield

File: polaris-icons/icons/Reports.yml [hunk 46]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
 authors:
-  - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Reports.yml:6]: Miru Alves

File: polaris-icons/icons/Reports.yml [hunk 47]
```diff
@@ -3,9 +3,9 @@ description: N/A
 keywords:
   - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/Reports.yml:4]: Report
Explore

File: polaris-icons/icons/ShippingLabel.yml [hunk 48]
```diff
@@ -0,0 +1,10 @@
+name: Shipping label
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ShippingLabel.yml:2]: Used to represent shipping labels.

File: polaris-icons/icons/ShippingLabel.yml [hunk 49]
```diff
@@ -0,0 +1,10 @@
...
+keywords:
+  - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ShippingLabel.yml:4]: Label
Shipping
Delivery
Package
Postage
Parcel

File: polaris-icons/icons/ShippingLabelFilled.yml [hunk 50]
```diff
@@ -0,0 +1,10 @@
+name: Shipping label filled
+description: N/A
+keywords:
+  - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ShippingLabelFilled.yml:4]: Label
Shipping
Delivery
Package
Postage
Parcel

File: polaris-icons/icons/ShippingLabelFilled.yml [hunk 51]
```diff
@@ -0,0 +1,10 @@
+name: Shipping label filled
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/ShippingLabelFilled.yml:2]: Used to represent shipping labels.

File: polaris-icons/icons/ShippingLabelFilled.yml [hunk 52]
```diff
@@ -0,0 +1,10 @@
...
+keywords:
+  - N/A
+authors:
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/ShippingLabelFilled.yml:6]: Holly Schofield

File: polaris-icons/icons/ShippingLabel.yml [hunk 53]
```diff
@@ -0,0 +1,10 @@
...
+authors:
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/ShippingLabel.yml:6]: Holly Schofield

File: polaris-icons/icons/Tax.yml [hunk 54]
```diff
@@ -7,7 +7,8 @@ keywords:
 authors:
   - Adam Whitcroft
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/Tax.yml:10]: Holly Schofield

File: polaris-icons/icons/TaxFilled.yml [hunk 55]
```diff
@@ -6,7 +6,8 @@ keywords:
   - tax
 authors:
   - Joe Thomas
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/TaxFilled.yml:9]: Holly Schofield

File: polaris-icons/icons/UsersAndPermissions.yml [hunk 56]
```diff
@@ -0,0 +1,10 @@
+name: Users and permissions
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/UsersAndPermissions.yml:2]: Used to represent access permissions for users.

File: polaris-icons/icons/UsersAndPermissionsFilled.yml [hunk 57]
```diff
@@ -0,0 +1,10 @@
+name: Users and permissions filled
+description: N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/UsersAndPermissionsFilled.yml:2]: Used to represent access permissions for users.

File: polaris-icons/icons/UsersAndPermissions.yml [hunk 58]
```diff
@@ -0,0 +1,10 @@
...
+keywords:
+  - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/UsersAndPermissions.yml:4]: Lock
Person
Access
Control
Rights
Roles
Security
Authentication
Authorization
Login
Password

File: polaris-icons/icons/UsersAndPermissionsFilled.yml [hunk 59]
```diff
@@ -0,0 +1,10 @@
...
+keywords:
+  - N/A
```
Comment (zecarlostorre) [inline polaris-icons/icons/UsersAndPermissionsFilled.yml:4]: Lock
Person
Access
Control
Rights
Roles
Security
Authentication
Authorization
Login
Password

File: polaris-icons/icons/UsersAndPermissionsFilled.yml [hunk 60]
```diff
@@ -0,0 +1,10 @@
...
+authors:
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/UsersAndPermissionsFilled.yml:6]: Holly Schofield

File: polaris-icons/icons/UsersAndPermissions.yml [hunk 61]
```diff
@@ -0,0 +1,10 @@
...
+authors:
+  - José Torre
```
Comment (zecarlostorre) [inline polaris-icons/icons/UsersAndPermissions.yml:6]: Holly Schofield

**Review by zecarlostorre — APPROVED** [review]
The icons look good, so I'm approving this.
If you want to include it, I added Holly's name to the icons she should be added to.
I also added descriptions and keywords to icons that didn't have them.

**Pull request merged**